src/OpenStack.Net/OpenStack/Services/ContentDelivery/V1/ContentDeliveryServiceExtensions.cs

```diff
@@ -0,0 +1,158 @@
+﻿namespace OpenStack.Services.ContentDelivery.V1
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using OpenStack.Collections;
+    using OpenStack.ObjectModel.JsonHome;
+    using Rackspace.Threading;
+
+    /// <summary>
+    /// This class provides extension methods that simplify the process of preparing and sending Content Delivery
+    /// Service HTTP API calls for the most common use cases.
+    /// </summary>
+    /// <threadsafety static="true" instance="false"/>
+    /// <preliminary/>
+    public static class ContentDeliveryServiceExtensions
+    {
+        public static Task<HomeDocument> GetHomeAsync(this IContentDeliveryService service, CancellationToken cancellationToken)
```

Comment:
:exclamation: This needs to be documented (along with several other cases in the same file).